README.md

```diff
@@ -1,2 +1,12 @@
 # fcrepo-module-auth-webac
-WebAC Authorization Delegate Module for the Fedora 4 Repository
+WebAC Authorization Delegate Module for the Fedora 4 Repository. This is an implementation of the W3C's WebAccessControl draft specification at: [https://www.w3.org/wiki/WebAccessControl](https://www.w3.org/wiki/WebAccessControl).
```

Comment:
Since this isn't even a draft specification, I wonder if we shouldn't use the phrase "proposed WebAccessControl" instead of "WebAccessControl draft specification"?